src/shared/loop-util.c

```diff
@@ -768,7 +768,12 @@ void loop_device_relinquish(LoopDevice *d) {
         d->relinquished = true;
 }
 
-int loop_device_open(const char *loop_path, int open_flags, LoopDevice **ret) {
+void loop_device_unrelinquish(LoopDevice *d) {
+        assert(d);
+        d->relinquished = false;
+}
+
+int loop_device_open(const char *loop_path, int open_flags, bool relinquish, LoopDevice **ret) {
```

Comment:
btw, why add this param? seems exotic enough so that calling loop_device_unrelinquish()/loop_device_relinquish() immediately after loop_device_open() returns should suffice, no?

Reply:
Removed